src/lib/components/artwork_grids/artwork.tsx

```diff
@@ -127,25 +129,41 @@ interface RelayProps {
     date: string | null,
     sale_message: string | null,
     is_in_auction: boolean | null,
-    sale_artwork: {
-      opening_bid: {
-        display: string | null,
-      } | null,
```

Comment:
Fix is on the way, sorry about that!

https://github.com/prettier/prettier/pull/1734